test/check-machines-consoles

```diff
@@ -44,13 +56,16 @@ class TestMachinesConsoles(machineslib.VirtualMachinesCase):
         b.wait_in_text(".pf-v5-c-console__manual-connection dl > div:nth-child(2) dd", "5900")
 
         b.click(".pf-v5-c-console__remote-viewer-launch-vv")  # "Launch Remote Viewer" button
-        b.wait_visible("#dynamically-generated-file")  # is .vv file generated for download?
```

Comment:
For context, this just checked that there is an HTML element which the `file-saver` library creates for downloading. As we now wait on the download this is not needed anymore and an implementation detail.

Reply:
This is actually not the `file-saver` library but our own code...